src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -153,15 +153,15 @@ class DropdownAutoCompleteMenu extends React.Component {
                   <div>
                     {menuHeader && <StyledLabel>{menuHeader}</StyledLabel>}
                     {this.autoCompleteFilter(items, inputValue).map(
-                      (item, index) =>
+                      ({index, ...item}) =>
```

Comment:
we want to use our special index that gets annotated in, not the one from map

Reply:
Oops I think I broke this, we should probably add a test case for this